Podfile

```
@@ -4,7 +4,7 @@
 target 'TodoApp' do
   # Comment the next line if you don't want to use dynamic frameworks
   use_frameworks!
-  pod 'RealmSwift'
+  pod 'RealmSwift', '5.4.0'
```

```suggestion
  pod 'RealmSwift', '~> 5.4.0'
```
でも良いと思います。

- 通常時はpod installを使う
    - メンバー間でのバージョン固定はPodfile.lockによってなされる
- アップデートしたい時は誰か一人がpod updateしてPodfile.lockをコミットする
    - `~> 5.4.0` と指定しておくと、pod updateした時に5.4.xの範囲でアップデートされる
    - バグフィックスがあった時には取り込みたいので、その観点で都合が良い
    - 現状でも既に5.4.3がリリースされている

CocoaPods の Podfile の書き方 - Notes | digitrick
https://dev.digitrick.us/notes/podfilesyntax

Tags · realm/realm-cocoa
https://github.com/realm/realm-cocoa/tags

---

Podfile.lockでバージョンを固定し、最新版を扱う場合はupdate。
これなら最新版への更新も可能ですね。
ありがとうございます！

---

ですです。チームメンバーにも周知しておくと良いですね。